### ISSUE TITLE
[0.0.42] ionic serve parameters not working

### ISSUE THREAD

**Issue by ohh2ahh** [driftyco/ionic-app-scripts#351]
#### Short description of the problem:
I upgraded to ionic-app-scrips 0.0.42 as described in the [CHANGELOG.md](https://github.com/driftyco/ionic-app-scripts/blob/master/CHANGELOG.md) and noticed, that `ionic serve --platform ios|android` and `ionic serve --lab` aren't working anymore (the URL is `http://localhost:8100` instead of `http://localhost:8100/?ionicplatform=ios`).

**Edit:** It seems that other parameters are affected as well (see comments below).

It works perfectly fine when I use the old scripts in `package.json`, even with ionic-app-scripts 0.0.42:
```
...
"scripts": {
    "build": "ionic-app-scripts build",
    "watch": "ionic-app-scripts watch",
    "serve:before": "watch",
    "emulate:before": "build",
    "deploy:before": "build",
    "build:before": "build",
    "run:before": "build"
    "ionic:build": "ionic-app-scripts build",
    "ionic:serve": "ionic-app-scripts serve"
  },
...
```
#### What behavior are you expecting?
`ionic serve --platform ios|android` and `ionic serve --lab` should open the browser with the desired platform(s).

**Which @ionic/app-scripts version are you using?**
0.0.42

**Other information:**
```
System information:

Cordova CLI: 6.4.0
Ionic Framework Version: 2.0.0-rc.2
Ionic CLI Version: 2.1.7
Ionic App Lib Version: 2.1.4
Ionic App Scripts Version: 0.0.42
ios-deploy version: 1.9.0
ios-sim version: 5.0.9
OS: macOS Sierra
Node Version: v7.0.0
Xcode version: Xcode 8.1 Build version 8B62
```

**Comment by konrin** [comment]
The parameter "-b" also stopped working

**Comment by chinib** [comment]
Parameter -c (consoleLogs) also

**Comment by Manduro** [comment]
You can use all the old parameters in your `package.json`'s scripts, like this:
```
...
"scripts" : {
  "ionic:build": "ionic-app-scripts build",
  "ionic:serve": "ionic-app-scripts serve -b -c"
}
...
```

All possible parameters are listed [here](https://github.com/driftyco/ionic-app-scripts/blob/master/src/serve.ts#L97).

**Comment by MELAS007** [comment]
all are stop, nothing working now!

**Comment by MikeLallemont** [comment]
We can no longer enter 'q' to stop the server. Must press ctrl-c. Is this by design?

**Comment by danbucholtz** [comment]
We are aware of these issues and are discussing how we want to move forward. We know we need to get `--lab` working and we are also working on getting live reload working on device working too.

Thanks,
Dan

**Comment by MELAS007** [comment]
@danbucholtz - localhost:8100/ionic-lab is not accessible anymore!

**Comment by jthoms1** [comment]
The latest CLI (2.1.8) should support passing of parameters such as --platform. Also the issues with lab are resolved in commit 1d99a9868edb11cc15b999ef6dda76c65f59b036.  In the mean time to access lab you can update the browser url location to be `/__ion-dev-server/ionic_lab.html`

**Comment by jthoms1** [comment]
We released a more recent CLI that fixes these issues.  Please ensure to install 2.1.12.

`npm install -g ionic@latest`
